docs/kb/wasm.rst

```diff
@@ -0,0 +1,307 @@
+=======================================
+Wasm support for user-defined functions
+=======================================
+
+This document describes the details of Wasm language support in user-defined functions (UDF). The language ``wasm`` is one of the possible languages to use, besides Lua, to implement these functions.
+
+
+.. note:: Until Scylla 5.2, the Wasm language is called ``xwasm``. This name will be replaced with ``wasm`` in Scylla 5.3.
```

Comment:
```suggestion
.. note:: Until ScyllaDB 5.2, the Wasm language is called ``xwasm``. The name is replaced with ``wasm`` in ScyllaDB 5.3.
```

Reply:
Let's not make the reference to the future as the documentation is going to be published in version 5.3.

Reply:
Ok